docs/references/users/create-token.md

```diff
@@ -0,0 +1 @@
+Returns a custom token with a secret key for creating a session. If the provided user ID has not be registered, a new user will be created. Use the returned user ID and secret and submit a request to the [PUT /account/sessions/custom](https://appwrite.io/docs/references/cloud/client-web/account#updateCustomSession) endpoint to complete the login process. By default, the secret sent in the response is valid for 15 minutes.
```

Comment:
I think endpoint is different, no more `custom`.
I don't think we need sentence about expiry, it will be there on the param

Reply:
Updated this